qrcode.js

```diff
@@ -0,0 +1,2136 @@
+//---------------------------------------------------------------------
```

Comment:
Do you know how this entire file works so that we can be sure that it does not contain any exploits that reveal a user's private keys?